### PR TITLE
Feat/dynamic art

### LINKED ISSUE #28: Create API route to retrieve album art from Spotify
**__Task Description__**
Describe the task to be completed.

**__Technical Considerations__**
Include any technical considerations including architecture (e.g. API), required libraries, etc.

**__Additional Considerations__**
Any supplemental information including unresolved questions, links to external resources, screenshots, etc.

### PR DESCRIPTION
**Pull request by Torja99**
Uses user's top tracks to display changing album art on quiz selection page.
Closes #28 #43 #44

### CODE REVIEW

File: hooks/useAuth.tsx [hunk 1]
```diff
@@ -53,18 +53,19 @@ function useProvideAuth() {
...
-  //     let fetchaAuth = await backendApi.get('/api/top/tracks');
-  //     console.log(fetchaAuth);
-  //   } catch (error: any) {
-  //     console.error('>>> ERR here', error);
-  //   }
-  // };
+  const testGetTopTracks = async () => {
+    try {
+      let fetchaAuth = await backendApi.get('/api/top/tracks');
+      console.log(fetchaAuth);
```
Comment (angelopoole) [inline hooks/useAuth.tsx]: Remember to remove logs, at least when merging into main

File: pages/index.tsx [hunk 2]
```diff
@@ -35,6 +36,7 @@ const Home: NextPage = () => {
         <QuizHero />
         <DataHero />
         <AboutHero />
+        <TestPage />
```
Comment (angelopoole) [inline pages/index.tsx]: ‼️ Make sure testing pages are not visible to end users  ‼️

File: todo.org [hunk 3]
```diff
@@ -0,0 +1,29 @@
+* TODO: hook up back end
```
Comment (angelopoole) [inline todo.org]: You probably don't want to commit your personal todo file up to github

**Review by angelopoole — CHANGES_REQUESTED** [review]
Some changes need to be made, Test page shouldnt show up in actual development, or at least should be really hard to get to within normal means, 

eg a user might not know that the route '/juniper/testPage' even exists.

There's a personal todo file committed and pushed here "todo.org"

File: hooks/useAuth.tsx (hunk 1 above)
Comment (Torja99) [reply hooks/useAuth.tsx]: For sure forgot about this thanks!

File: pages/index.tsx (hunk 2 above)
Comment (Torja99) [reply pages/index.tsx]: oops forgot to remove

File: todo.org (hunk 3 above)
Comment (Torja99) [reply todo.org]: Right removed it

**Pull request merged**